onadata/apps/api/viewsets/floip_viewset.py

```diff
@@ -57,11 +90,19 @@ class FloipViewSet(mixins.CreateModelMixin, mixins.DestroyModelMixin,
     queryset = XForm.objects.filter(deleted_at__isnull=True)
     serializer_class = FloipSerializer
 
-    pagination_class = PageNumberPagination
+    pagination_class = FLOIPResponsePageNumberPagination
     parser_classes = (JSONParser, )
     renderer_classes = (FlowResultsJSONRenderer, )
 
     lookup_field = 'uuid'
+    filter_map = {
+        "filter[max-version]": "version__lte",
+        "filter[min-version]": "version__gte",
+        "filter[start-timestamp]": "date_created__gte",
+        "filter[end-timestamp]": "date_created__lte",
+        "page[afterCursor]": "id__gt",
+        "page[beforeCursor]": "id__lt",
```

Comment:
We might have implemented this in a way that doesn't conform with the FLOIP Spec. The specification states

```
"page[afterCursor]: The response row_id to requests responses after this id, when paginating forward"
"page[beforeCursor]: The response row_id to request responses prior to this id, when paginating in reverse "
```

The `row_id` seems to be something we generate when we create the FLOIP list [here](https://github.com/onaio/onadata/blob/master/onadata/libs/renderers/renderers.py#L61)